docs/tutorials/how_to_new_pokemon_1_10_0.md

```diff
@@ -0,0 +1,1153 @@
+    - Because you are limited to two frames, there are already [predefined front sprite animations](https://github.com/rh-hideout/pokeemerald-expansion/blob/master/include/pokemon_animation.h), describing translations, rotations, scalings or color changes.
+- `frontAnimDelay`:
+    - Sets a delay in frame count between when the Pokémon appears and when the animation starts.
+- `enemyMonElevation`:
+    - Used to determine the altitude from the ground. Any value above 0 will show a shadow under the Pokémon, to signify that they're floating.
+- `backPic`:
+    - Used to reference the back sprite, so in this case, we call for `gMonBackPic_Mewthree`.
+- `backPicSize`:
+    - The two values (`width` and `height`) are used for defining the non-empty size of the back sprite, which is used in move animations. If you're unsure of the values, you can leave them both as 64.
+    - **NOTE**: Mew has a tarnary switch here in order to change values depending on if a config option is set for displaying th original Gen 3 sprites.
+- `backPicYOffset`:
+    - Used to define what Y position of the back sprite. When working with the animation debug menu, we recommend aligning the back sprite to the white background, as it was designed to properyly align with the real battle layout.
+- `backAnimId`:
+    - Like `frontAnimId` except for the back sprites and them being a single frame. The IDs listed [here](https://github.com/rh-hideout/pokeemerald-expansion/blob/master/include/pokemon_animation.h) are used to represent 3 different animations that happen based on the the Pokémon's nature.
+- `palette`:
+    - Used to reference the non-shiny palette, so in this case, we call for `gMonPalette_Mewthree`.
+- `shinyPalette`:
+    - Used to reference the shiny palette, so in this case, we call for `gMonShinyPalette_Mewthree`.
+- `iconSprite`:
+    - Used to reference the icon sprite, so in this case, we call for `gMonIcon_Mewthree`.
+- `iconPalIndex`:
+- `ICON`
+    - Here, you can choose between the six icon palettes; 0, 1, 2, 3, 4 and 5. All of them located in `graphics/pokemon/icon_palettes`.
+
+        Open an icon sprite and load one of the palettes to find out which palette suits your icon sprite best.
+- `FOOTPRINT`
+    - We made this single field into a macro so that they can be ignored when `P_FOOTPRINTS` is set to false. It's also why we don't have an "," after calling it like the other macros (we add it as part of the macro itself).
+        ```c
+        #if P_FOOTPRINTS
+        #define FOOTPRINT(sprite) .footprint = gMonFootprint_## sprite,
+        #else
+        #define FOOTPRINT(sprite)
+        #endif
+        ```
+
+# The Data - Part 2
+
+We're almost there just a bit left!
+
+## 1. Species Flags
+
+```diff
+ const struct SpeciesInfo gSpeciesInfo[] =
+ {
+     ...
+     [SPECIES_MEWTHREE] =
+     {
+        ...
+        .abilities = { ABILITY_INSOMNIA, ABILITY_NONE, ABILITY_NONE },
+        .bodyColor = BODY_COLOR_PURPLE,
++       .isLegendary = TRUE,
```

Comment:
None of these flags are listed in the example template in `species_info.h`, not sure whether or not we want to add them (it seems they haven't been added in the past, figured it's worth considering whether we want to continue to leave them out)

Reply:
Their purpose is listed further down, and the example is specifying an evolution to Mew that's legendary, so either the example should change, or this stays.

Reply:
I don' think anything in the document needs to change, what I meant is that if it were me I'd probably add all of these flags as `.isLegendary =FALSE` etc. at the bottom of the example in `species_info.h` so users have all of the options available when they copy paste the template. I don't think anything about the guide itself needs to change, I mean adding the parameters to the `species_info.h` template. I'm okay to leave it as is if you disagree though :)